test/smoke_test.py

```diff
@@ -79,6 +80,18 @@ def main() -> None:
     print(f"torchvision: {torchvision.__version__}")
     print(f"torch.cuda.is_available: {torch.cuda.is_available()}")
 
+    if re.match(r"\d+\.\d+\.\d+(?!a0)", torchvision.__version__):
+        try:
+            import torchvision.prototype as _
```

Comment:
We need the `as _`, because otherwise the import interferes with the regular `import torchvision` at the top of the module.

Reply:
```suggestion
            from torchvision import prototype
```

Reply:
:facepalm: